Pull request checklist (#39)
Skittyblock/aidoku-community-sources: .github/PULL_REQUEST_TEMPLATE.md

Checklist:
- [ ] Updated source's version for individual source changes
- [ ] Updated all sources' versions for template changes
- [ ] Set appropriate `nsfw` value

> - [ ] Have kept `id` if a source's name or language were changed
> - [ ] Have tested the modifications by running it on the simulator or a test device 

Reviewer: ```suggestion
- [ ] Did not change `id` even if a source's name or language were changed
- [ ] Tested the modifications by running it on the simulator or a test device 
```
